bootstrap/container/logging.go

```diff
@@ -25,5 +25,9 @@ var LoggingClientInterfaceName = di.TypeInstanceToName((*logger.LoggingClient)(n
 
 // LoggingClientFrom helper function queries the DIC and returns the logger.loggingClient implementation.
 func LoggingClientFrom(get di.Get) logger.LoggingClient {
-	return get(LoggingClientInterfaceName).(logger.LoggingClient)
+	loggingClient := get(LoggingClientInterfaceName)
+	if loggingClient != nil {
```

Comment:
suggested change

```
if loggingClient, ok := get(LoggingClientInterfaceName).(logger.LoggingClient); ok {
  return loggingClient
} else {
  return nil
}

```

Reply:
Awesome!  revised the codes per your suggestion.